virtualenv.py

```diff
@@ -1580,6 +1580,20 @@ def fix_lib64(lib_dir, symlink=True):
         else:
             copyfile('lib', lib64_link)
 
+def resolve_exe(exe):
+    if os.path.exists(exe):
+        return exe
+
+    if os.name != 'nt':
+        return None
+
+    # check PATHEXT
+    exts = os.environ['PATHEXT'].split(os.pathsep)
```

Comment:
Do we really want to use ```PATHEXT``` here? If what you've found isn't ```python.exe``` it's unlikely to actually be a valid Python installation, after all...